synchronizations/Synchronization.md

```diff
@@ -0,0 +1,223 @@
+            var personRootModel = new Model();
+
+            familyRegister = new FamilyRegister();
+            personRegister = new PersonRegister();
+
+            synchronization.Synchronize(
+                synchronization.SynchronizationRule<Families2PersonsSynchronization.FamilyRegisterToPersonRegister>(),
+                ref familyRegister, // Familyregister is the left model
+                ref personRegister, // Personregister is the right model
+                SynchronizationDirection.LeftWins, // prioritizes the left synchronization of left model (without shouldCorrespond it has no impact)
+                ChangePropagationMode.TwoWay); // TwoWay means Bidirectional synchronization 
+
+            familyRootModel.RootElements.Add(familyRegister);
+            personRootModel.RootElements.Add(personRegister);
+
+            repository.Models.Add(new Uri("ttc:source"), familyRootModel);
+            repository.Models.Add(new Uri("ttc:target"), personRootModel);
+        }
+```
+
+Here, we are creating two instances of the [Model](../models/api/NMF.Models.Model.yml) class. These instances, `familyRootModel` and `personRootModel`, are initially empty. In our case, these models represent the root nodes in a tree-like structure.
+
+We then create instances of `FamilyRegister` and `PersonRegister`. These registers are child nodes of the models, and they manage specific types of data, in our case Familymembers and Persons:
+- `FamilyRegister`: As the name suggests, this register manages families. A family, in this context, consists of a parent couple (mother and father) and can have children.
+- `PersonRegister`: This register manages individual persons. These persons form the foundation for families, as each family is composed of individuals.
+
+
+```csharp
+synchronization.Synchronize(
+                synchronization.SynchronizationRule<Families2PersonsSynchronization.FamilyRegisterToPersonRegister>(), // synchronization block/rule
+                ref familyRegister, // Familyregister is the left model
+                ref personRegister, // Personregister is the right model
+                SynchronizationDirection.LeftWins, // prioritizes the left synchronization of left model (without shouldCorrespond it has no impact)
+                ChangePropagationMode.TwoWay); // TwoWay means Bidirectional synchronization
+```                
+Synchronization Method Call:
+    The `Synchronize()` method is invoked on the synchronization object to establish synchronization between two models.
+    This method configures how data is synchronized between the models.
+Synchronization Rule:
+    The synchronization rule, `FamilyRegisterToPersonRegister`, is specified from the `Families2PersonsSynchronization` class.
+    It defines the logic for mapping data between the family register and the person register.
+Adding Registers:
+    Both the family register and the person register are passed as parameters to the Synchronize method.
```

Comment:
The idea is that you declare that these two model elements are corresponding to each other